shim/src/optimizer.js

```diff
@@ -51,20 +61,27 @@ const keywords = new Set([
   'with',
   'yield',
 
-  // future reserved word
+  // Also reserved when parsing strict mode code
+  'let',
+  'static',
+
+  // 11.6.2.2 Future Reserved Words
   'enum',
 
-  // future reserved word in strict mode
+  // Also reserved when parsing strict mode code
   'implements',
   'package',
   'protected',
   'interface',
   'private',
   'public',
 
-  // contextual worth refusing
-  'let',
-  'async',
```

Comment:
I think we should continue to omit `async` or anything similarly almost-reserved. It costs little to omit these, since no one will use them anyway. By omitting them, we have fewer worries we need to think about.

We should also omit `arguments` on the same grounds. See https://github.com/Agoric/TinySES/blob/master/src/tinyses.js#L49

Reply:
Ok, re-omitting `async`, maybe those should be part of the specs?
FYI `arguments` is already omitted, and we have a test for it.